lib/ansible/modules/network/solace/solace_dmr.py

```diff
@@ -94,21 +95,24 @@
     type: dict
 '''
 
-import ansible.module_utils.network.solace.solace_utils as su
-from ansible.module_utils.basic import AnsibleModule
-
 
 class SolaceDMRClusterTask(su.SolaceTask):
 
+    LOOKUP_ITEM_KEY = 'dmrClusterName'
```

Comment:
Can this be the standard approach - define the `LOOKUP_ITEM_KEY` as the first line of the class - it makes it easier to find :)